lib/core/uploads/MongoDBUploadsRepository.ts

```diff
@@ -13,4 +14,8 @@ export class MongoDBUploadsRepository implements UploadsRepository {
   async deleteManyByUuids(uuids: Upload['uuid'][]): Promise<void> {
     await this.model.deleteMany({ uuid: { $in: uuids } });
   }
+
+  async create(upload: Omit<Upload, 'id'>) {
```

Comment:
`async-await` could be removed from here

Reply:
fixed